.github/workflows/linters.yml

```diff
@@ -2,51 +2,25 @@ name: Linters
 
 on: pull_request
 
+env:
+  FORCE_COLOR: 1
+
 jobs:
   rubocop:
     name: Rubocop
     runs-on: ubuntu-22.04
-    
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-ruby@v1
         with:
-          ruby-version: ">=3.1.x"
+          ruby-version: 3.1.x
       - name: Setup Rubocop
         run: |
           gem install --no-document rubocop -v '>= 1.0, < 2.0' # https://docs.rubocop.org/en/stable/installation/
-          [ -f .rubocop.yml ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/ruby/.rubocop.yml
+          [ -f .rubocop.yml ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/ror/.rubocop.yml
       - name: Rubocop Report
         run: rubocop --color
 
-  lighthouse:
-    name: Lighthouse
-    runs-on: ubuntu-22.04
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
-        with:
-          node-version: "18.x"
-      - name: Setup Lighthouse
-        run: npm install -g @lhci/cli@0.11.x
-      - name: Lighthouse Report
-        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
-
-  stylelint:
-    name: Stylelint
-    runs-on: ubuntu-22.04
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
-        with:
-          node-version: "18.x"
-      - name: Setup Stylelint
-        run: |
-          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
-          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
-      - name: Stylelint Report
-        run: npx stylelint "**/*.{css,scss}"
-
```

Comment:
- [ ] [OPTIONAL]
Stylint and Webhint are not required for this project, however, you should uninstall them completely if mistakenly installed instead of removing them from the configuration file.